lazer/evm/src/ExampleReceiver.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Why do we need to do this?

Reply:
As the license for the repo is Apache License. Hence updated here.

For reference https://github.com/OpenZeppelin/openzeppelin-contracts/blob/49882a3d8b6c48068d9c988876d164b48dde5ca3/contracts/utils/math/Math.sol#L1C1-L1C32